src/test/java/com/rtb/projectmanagementtool/auth/AuthOpsTest.java

```diff
@@ -75,60 +83,43 @@ public void testAlreadyLoggedInDontSetCookie() {
 
   @Test
   public void testLoginUser() {
-
-    System.out.println("Checkpoint 1");
+    // auth object
+    ds = DatastoreServiceFactory.getDatastoreService();
```

Comment:
class DatastoreService, DataServiceFactory, ds not recognized and can't figure out why ... missing imports?

Reply:
Are you missing the type? `DatastoreService ds = ...`